lib/controller.js

```diff
@@ -19,8 +20,8 @@ controller.getPairing = (request, response) => {
 controller.getActions = (request, response) => {
     ActionService.getActions()
         .then((actions) => response.json(actions))
-        .catch(() => {
-            response.status(503).json({message: 'Unable to retrieve actions'});
+        .catch((error) => {
+            response.status(error.statusCode).json({message: error.message});
```

Comment:
you are doing a lot of changes that make debug information transparent. 503 means service unavailable. That decision was made so it displays that when the core is not working properly
let's discuss it a bit

Reply:
Actually we are making sure that SDK does not mask any errors return from Core with these changes. One of the client is not happy with the return value of 503 always for all kinds of error situations, hence these changes to SDK.